app/src/test/java/uk/gov/onelogin/MainActivityViewModelTest.kt

```diff
@@ -57,13 +41,9 @@ class MainActivityViewModelTest {
     )
 
     private val viewModel = MainActivityViewModel(
```

Comment:
The system under test (MainActivityViewModel) should be recreated each fresh for each test, so that any possible state is not accidentally retained between tests. This is why it is still acceptable to use `lateinit var` within test classes, when it has fallen out of favour for production code.

Reply:
done